### PR TITLE
[ci] no conda

### PR DESCRIPTION
**Pull request by danking**
Remove conda from ci. Mirrors the changes in batch.

I already tested deployment works by hand deploying a bogus version. It didn't have any watched targets though, so I didn't test everything. The pod starts successfully in python 3.6 (ci previously used python 3.7, see changes in shell_helper.py).

### CODE REVIEW

File: ci/Makefile [hunk 1]
```diff
@@ -61,22 +60,22 @@ run-local: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test
...
+	python3 run_ci.py
 
 run-local-for-tests: HAIL_CI_REMOTE_PORT = 3001
 run-local-for-tests: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
 run-local-for-tests: restart-all-proxies
 	SELF_HOSTNAME=http://${PROXY_IP}:${HAIL_CI_REMOTE_PORT} \
 	BATCH_SERVER_URL=http://127.0.0.1:${HAIL_CI_LOCAL_BATCH_PORT} \
 	WATCHED_TARGETS='[["hail-is/ci-test:master", true]]' \
-	. ../loadconda && conda activate hail-ci && pip install ./batch && python run_ci.py
+	pip3 install ../batch && python run_ci.py
```
Comment (jigold) [inline ci/Makefile]: python3 run_ci.py

File: ci/ci/shell_helper.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 
 def shell(*args):
-    sp.run(args, capture_output=True, check=True)
+    sp.run(args, stderr=sp.PIPE, stdout=sp.PIPE, check=True)
```
Comment (jigold) [inline ci/ci/shell_helper.py]: What's the reason for this change?

Comment (danking) [reply ci/ci/shell_helper.py]: Python 3.7 added capture_output. We're reverting to python 3.6.

Comment (jigold) [reply ci/ci/shell_helper.py]: hmmm... that must mean pipeline will eventually need to be updated.

File: ci/test-locally.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-python3 -m pip --user install -U ../batch
+python3 -m pip install --user -U ../batch
```
Comment (jigold) [inline ci/test-locally.sh]: Does --user need to be in the Dockerfile as well?

Comment (danking) [reply ci/test-locally.sh]: nah that happens in a docker image with root privileges. our tests don't run as root.
